gwardia/templates/gwardia/index.html

```diff
@@ -0,0 +1,7 @@
+{% extends 'core/base.html' %}
```

Comment:
`core/base.html` is just black background now, so `gwardia/index.html` is black background with text in left upper corner, did you want this to look like this??

![image](https://github.com/Gwardia-Czapli/GwardiaHub/assets/88268063/f8413ce8-bbeb-4301-894b-c01bb3241fe1)

Reply:
Yes, because `core.html` adds a lot of important things like title, scripts (in future also open graph tags). It is just WIP, we can improve `/gwardia` in future.